testsuite/N807.py

```diff
@@ -1,3 +1,22 @@
+#: Okay
+class C1:
+    def __str__(self):
+        return ''
+#: Okay
+class C2:
+    if True:
+        def __str__(self):
+            return ''
+#: Okay
```

Comment:
Is there an example where we'd want to report an error, e.g., N807 within a conditional? If so, can you add another test for that here, please?

Reply:
Done!